client/foundries.go

```diff
@@ -1048,8 +1048,11 @@ func (a *Api) TufProdRootPost(factory string, root []byte) (string, error) {
 	return a.tufRootPost(factory, true, root)
 }
 
-func (a *Api) TargetsListRaw(factory string) (*[]byte, error) {
+func (a *Api) TargetsListRaw(factory string, productionTag string) (*[]byte, error) {
 	url := a.serverUrl + "/ota/repo/" + factory + "/api/v1/user_repo/targets.json"
+	if len(productionTag) > 0 {
+		url += "?production=1&tag=" + productionTag
```

Comment:
Oops!  We have a production targets API at this URL: "/ota/repo/" + factory + "/prod-targets/?tag=" + productionTag.

And I think it'd be good to have a `ProdTargetsList` function which works with it.

Reply:
force push fix